css/src/reset.css

```diff
@@ -16,11 +16,10 @@ article, aside, canvas, details, embed,
 figure, figcaption, footer, header, hgroup,
 menu, nav, output, ruby, section, summary,
 time, mark, audio, video {
-  margin: 0;
+    margin: 0;
 	padding: 0;
-	border: 0;
+    border: 0;
 	font-size: 100%;
-	font: inherit;
```

Comment:
Right now, this fix was required for supporting markdown support in PropTable of docPage. Since, we were using font: inherit, font styling of storybook addon-docs gets updated.

Reply:
I am closing this pull request as it will be fixed by decoupling the reset css from the library css.

Reply:
Ok. Sounds good.